samples.http

```diff
@@ -28,7 +28,8 @@ Content-Type: application/json
 
 
 ### Add highlight to existing match
-POST {{baseUrl}}/matches/1/highlights HTTP/1.1
+### 60f81c4755c33a09f2958b86
+POST {{baseUrl}}/matches/60f81c4755c33a09f2958b86/highlights HTTP/1.1
```

Comment:
We might have different databases and the id only exists in your environment.
The REST client might have a solution for this in the extension documentation.

Reply:
thanks, fixed